ui/packages/catalog-ui-search/src/main/webapp/react-component/location/location.js

```diff
@@ -103,17 +145,63 @@ const ddValidators = {
   east: value => value <= 180 && value >= -180,
 }
 
+let isDms = false
+const dmsValidators = {
+  dmsLat: value => validateInput(value, 'dd°mm\'ss.s"'),
+  dmsLon: value => validateInput(value, 'ddd°mm\'ss.s"'),
+  dmsNorth: value => validateInput(value, 'dd°mm\'ss.s"'),
+  dmsSouth: value => validateInput(value, 'dd°mm\'ss.s"'),
+  dmsWest: value => validateInput(value, 'ddd°mm\'ss.s"'),
+  dmsEast: value => validateInput(value, 'ddd°mm\'ss.s"'),
+}
+
+const getNegOrPosLatLon = (key, value) => {
```

Comment:
getLatLonAbsoluteValue ?

Reply:
It isn't the absolute value for the coordinates. In the case of point radius the UI doesn't present an option for east/west or north/south so you need to enter a number from -90 - 90 or -180 - 180. This is for the error message and whether or not to default to 90 or -90 based on what the user put in the text box

Reply:
validLatLon might be better named as defaultLatLon